Demo/DemoViewController.swift

```diff
@@ -37,8 +37,8 @@ class DemoViewController: UIViewController, CLLocationManagerDelegate {
   private let regionSize: CLLocationDistance = 500.0
   private let distanceFilter: CLLocationDistance = 10.0
 
-  var demoView: DemoView {
-    return view as! DemoView
+  var demoView: DemoView? {
```

Comment:
👏 I only starting avoiding force unwraps recently, three years after I started writing Swift. I'm still not convinced they are inherently bad, but they are so controversial that I believe it is better to avoid them.

When I use an optional property repeatedly in a function, I sometimes `guard` at the start of the function, but that is not necessary in this case, mostly because `demoView` is not used as an argument to another function.

Reply:
Yea, I understand what you mean! My biggest argument against them only fits in production code at the end of the day. Essentially to me force unwraps are a quick "let's make this easier on ourselves" when building something quickly, and they work against Swift's built in strength and measures to make sure we are diligent about the actuality of state and how it can be unexpected, especially when lots of hands are touching a product or I get forgetful ;)